brayns/common/scene/Scene.h

```diff
@@ -150,27 +157,54 @@ class Scene
     /**
         Return true if the scene does not contain any geometry. False otherwise
     */
-    BRAYNS_API bool isEmpty() { return _isEmpty; }
+    BRAYNS_API bool getEmpty() { return _primitives.size() == 0 && _trianglesMeshes.size() == 0; }
```

Comment:
I prefer isEmpty() for boolean functions like this, getEmpty() gives me the impression you are trying to get an empty unspecified thing. This function is const, too.